alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
I'm a bit confused why this service is so complex. Can't we have something simpler like this? https://aur.archlinux.org/cgit/aur.git/tree/ModemManager?h=modemmanager-openrc